credential-stores/postgres/main.go

```diff
@@ -28,28 +28,29 @@ func main() {
 	}
 
 	mux := http.NewServeMux()
-	mux.HandleFunc("/", func(w http.ResponseWriter, _ *http.Request) {
-		w.WriteHeader(http.StatusOK)
-	})
-	mux.HandleFunc("/store", func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc("/store", authenticatedHandler(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Can we make all of these accept only `POST` requests? GPTScript will only send `POST` requests to tools.

Reply:
fixed